lib/wca_live_web/router.ex

```diff
@@ -48,6 +48,8 @@ defmodule WcaLiveWeb.Router do
     pipe_through :api
 
     get "/competitions/:id/wcif", CompetitionController, :show_wcif
+    get "/competitions/:id/results", CompetitionController, :show_results
+    get "/competitions/:wca_id/id", CompetitionController, :show_competition_id
```

Comment:
```suggestion

    # Public results endpoint
    get "/competitions/:id_or_wca_id/results", CompetitionController, :show_results
```

If the idea is to request the id just to make the results request, I would rather make it `/competitions/:id_or_wca_id/results`. We can do a similar logic as here:

https://github.com/thewca/wca-live/blob/7c01c5a94b04ddad49758c5b441d2be1c683ec69/lib/wca_live_web/controllers/link_controller.ex#L27-L44

We can determine which id we are dealing with, because one is integer, and the other is not.